skactiveml/pool/_wrapper.py

```diff
@@ -0,0 +1,106 @@
+"""
+Wrapper for skactiveml PoolQueryStrategy to deal utility weights.
+"""
+
+# Authors: Pascal Mergard <Pascal.Mergard@student.uni-kassel.de>
+
+from sklearn.utils import check_array
+from ..base import PoolQueryStrategy
+from ..utils import rand_argmax
+
+
+class UtilityWrapper(PoolQueryStrategy):
+    """UtilityWrapper
+
+    Implementation of a wrapper class for skactiveml query strategies to deal
+    utility weights.
+
+    Parameters
+    ----------
+    query_strategy : skaktiveml.PoolQueryStrategy
+        The query strategy to wrap.
+    """
+    def __init__(self, query_strategy):
```

Comment:
add self.weight_utilities_function to init with None as default:

default (self.weight_utilities_function = None) function should look like this:
```
def mult_utilities_weight(utilities, utility_weight):
    if np.any(utilities < 0):
        raise ValueError("")
    # add_optional_checks
    return utilities * utility_weight
```